docs/user-guide/grpc.md

```diff
@@ -92,14 +92,24 @@ So we modify the "gRPC server example" to use our own self-signed certificate:
 ```go
 // ...
 
+import (
```

Comment:
Could you remove this block (imports) ?

Reply:
with your modification we need to add some import

Reply:
please remove